mutiny-core/src/nodemanager.rs

```diff
@@ -446,6 +446,9 @@ impl NodeManager {
             return Err(MutinyError::WalletOperationFailed);
         };
 
+        // if empty string, treat as None
+        let description = description.filter(|d| !d.is_empty());
+
         // TODO if there's no description should be something random I guess
         let invoice = self
             .create_invoice(amount, description.clone().unwrap_or_else(|| "".into()))
```

Comment:
Isn't this just turning it back into `""` anyways? I feel like the issue is more downstream than here.

Reply:
Yeah but this will make it so the description on what we return is `None`.

Also I have a todo of `/ TODO if there's no description should be something random I guess`, which we'll want to handle if the description is none

Reply:
I feel that it is better to fix it closer to the source of the problem than above here. So instead of it failing if every call to `create_invoice` is an empty string, have it treat empty string and `None` as the same inside of `create_invoice`, then any call to `create_invoice` won't have the same fate.

Reply:
Good call, I like this a lot better